ceph/iscsi/gateway.py

```diff
@@ -0,0 +1,183 @@
+"""iSCSI GWCLI Library Module.
```

Comment:
The current implementation has multiple nested classes. I strongly beleive we could use a different approach. As some of the class behaviors are similar (create, delete), a base class could help improve reusability.